report_templates/entitlements.erb

```diff
@@ -2,17 +2,25 @@
 name: Subscription - Entitlement Report
 snippet: false
 model: ReportTemplate
+template_inputs:
+- name: days_from_now
+  required: false
+  input_type: user
+  description: "Show subscriptions expiring in <= this many days.  Leave blank to show all entitlements."
+  advanced: false
+  value_type: plain
+  options: "7\r\n30\r\n60\r\n90\r\n120\r\n180"
 require:
 - plugin: katello
   version: 3.14.0
 -%>
-<%- report_headers 'Name', 'Organization', 'Lifecycle Environment', 'Content View', 'Host Collections', 'Virtual', 'Guest of Host', 'OS', 
```

Comment:
Why do you remove this? Headers are required, in case of `load_hosts` returns zero objects, the result would be empty report. However even empty dataset should contain the structure. Empty csv file caused confusion in the past.

Reply:
@ares I think I removed it by mistake when copying the ERB from the entitlements report in the web UI editor (it doesn't show there for some reason.) It seemed to work fine so I didn't catch it.  Will add it back.